src/main/java/uk/gov/pay/adminusers/model/Link.java

```diff
@@ -37,6 +38,11 @@ private Link(@JsonProperty("rel") Rel rel,
         this.href = href;
     }
 
+    @JsonProperty("rel")
+    public String getRelAsLowerCase() {
+        return rel.name().toLowerCase(Locale.UK);
```

Comment:
`Locale.ENGLISH` is probably slightly more appropriate for this purpose (our lower-casing is English-specific but not UK-specific)

Reply:
👍 done.